src/components/Form/CrossPodWriteForm.jsx

```diff
@@ -97,51 +107,106 @@ const CrossPodWriteForm = () => {
       statusType="Upload status"
       defaultMessage="To be uploaded..."
     >
-      <form onSubmit={handleCrossPodUpload} autoComplete="off">
-        <div style={formRowStyle}>
-          <label htmlFor="cross-upload-doc">Upload document to username: </label>
-          <br />
-          <br />
-          <input
-            id="cross-upload-doc"
-            size="25"
-            name="crossPodUpload"
-            value={username}
-            onChange={(e) => setUsername(e.target.value)}
-            placeholder={selectedUser.username}
-          />
-        </div>
-        <div style={formRowStyle}>
+      <Box display="flex" justifyContent="center">
+        <form onSubmit={handleCrossPodUpload} autoComplete="off">
+          <FormControl fullWidth>
+            {/* TODO: Determine whether Typography below is necessary or redundant */}
+            {/* Either way, also determine whether to update all other forms to that */}
+            <Typography htmlFor="cross-upload-doc">Upload document to username:</Typography>
+            <TextField
+              id="cross-upload-doc"
+              name="crossPodUpload"
+              label="Search Username"
+              required
+              value={username}
+              onChange={(e) => setUsername(e.target.value)}
+              placeholder={selectedUser.username}
+            />
+          </FormControl>
           <DocumentSelection
             htmlForAndIdProp="upload-doc"
             handleDocType={handleDocType}
             docType={docType}
           />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doc-expiration">Expiration date (if applicable): </label>
-          <input id="upload-doc-expiration" name="date" type="date" />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doc-desc">Enter description: </label>
           <br />
+          <FormControl>
+            {/* TODO: Determine whether InputLabel below is necessary or redundant */}
+            <InputLabel htmlFor="upload-doc-expiration" />
```

Comment:
Think we can remove these Lines for 133 to 134 without impacting functionality.

That means we can also remove the `id` attribute in Line 137.